src/applications/medical-copays/containers/HTMLStatementPage.jsx

```diff
@@ -0,0 +1,84 @@
+import React, { useEffect } from 'react';
+import { useSelector } from 'react-redux';
+import { Link } from 'react-router-dom';
+import scrollToTop from 'platform/utilities/ui/scrollToTop';
+import PropTypes from 'prop-types';
+import moment from 'moment';
+import Modals from '../components/Modals';
+import StatementAddresses from '../components/StatementAddresses';
+
+const HTMLStatementPage = ({ match }) => {
+  const selectedId = match.params.id;
+  const statements = useSelector(({ mcp }) => mcp.statements) ?? [];
+  const [selectedCopay] = statements?.filter(({ id }) => id === selectedId);
```

Comment:
So if statements are null, nothing on the page is going to be showing correctly and/or be useful? 

The question(s) here are 

- Do we care if `statements` is null/undefined and what does that mean for our page?
- Do we care if `selectedCopay` is null/undefined and what does that mean for the page?

`?.` can easily swallow errors, I tend to err on the side of `fail early and fail fast`. 

Nothing blocking this PR, but going forward we might want to look at better handling these checks. Whether is is showing an error page or always expecting these things to be there.